lemongrab/static/js/todo.js

```diff
@@ -1,132 +1,118 @@
-'use-strict';
-
 TODO = (function(todoCode) {
 
   return todoCode(window.jQuery, window, document);
 
 }(function($, window, document) {
-  // CSS selectors values and other global constants
-  var constants = {
-    task: "task",
-    taskDescription: "task-body",
-    control: "task-control",
-    completeAction: "complete-task",
-    completeText: "Complete",
-    removeAction: "delete-task",
-    removeText: "Remove",
-    taskIdPrefix: "task-",
-    taskInput: "new-task-form",
-    noNameError: "Name cannot be empty",
-    incompleteList: "#incomplete",
-    completeList: "#complete"
-  };
+  "use strict";
+
+  // Bind events to the DOM
+  var _bind_events = function() {
+
+    // Register event handler for 'enter' keypress
+    $('#new-task-form').keypress(function(event){
+
+      if (event.keyCode == 10 || event.keyCode == 13) {
+        event.preventDefault();
+        $('#new-task-form .button').click();
+      }
+
+    });
+
+  }
 
   // The DOM is ready
-  $(function() { /* We currently do not perform any setup */});
+  $(function() {
+    _bind_events();
+
+  });
 
-  // Create the task html and add it to the To Do list
+  // Create the task and add it to the To Do list
   var _createTask = function(id, description) {
-    var parent = $(constants.incompleteList),
-        task, control;
+    var parent  =   $("#task-list"),
```

Comment:
The style guide I mentioned had a tab before and after the "=" for all variables. We're going to be declaring every variable with a single var statement, so we're trying to mak it visuall easy to scan the full list of them. The tabs are to align the variable names and instantiations vertically. Of course, I don't have any methods that have multiple variables instantiated at the top. 

Style of course is a preference and we don't need to use that. Or we could only apply it when there are multiple variable instantiations. Would you prefer that?